examples/Model.elm

```diff
@@ -364,7 +392,7 @@ fetchRecordList model =
             |> Kinto.getList recordResource
             |> Kinto.sortBy [ sortColumn ]
             |> limiter
-            |> Kinto.send FetchRecordsResponse
+            |> Kinto.send (FetchRecordsResponse False)
```

Comment:
I'm not sure I understand the use for `append`, why do we need the `False` here? Wouldn't it just append to the empty list of `model.records`?